plugins/woocommerce/tests/e2e-pw/utils/tours.js

```diff
@@ -0,0 +1,31 @@
+const { admin } = require( '../test-data/data' );
+
+const base64String = Buffer.from(
+	`${ admin.username }:${ admin.password }`
+).toString( 'base64' );
+
+const headers = {
+	Authorization: `Basic ${ base64String }`,
+	cookie: '',
+};
+
+/**
+ * Enables or disables the product editor tour.
+ *
+ * @param {import('@playwright/test').APIRequestContext} request Request context from calling function.
+ * @param {boolean} enable Set to `true` if you want to enable the block product tour. `false` if otherwise.
+ */
+const toggleBlockProductTour = async ( request, enable ) => {
+	const url = '/wp-json/wc-admin/options';
+	const params = { _locale: 'user' };
+	const toggleValue = enable ? 'no' : 'yes';
```

Comment:
Here I understand that if the function argument **enable** is `true`, then `woocommerce_block_product_tour_shown` will be set to `no`, which sounds different than the explanation of the argument in the function documentation:
```
@param {boolean} enable Set to `true` if you want to enable the block product tour. `false` if otherwise.
```

Could I be missing anything here?

Reply:
Right, it does appear confusing at first. But the documentation and the logic is still correct. 

When the `woocommerce_block_product_tour_shown` property is set to `no`, that means the tour was **not** shown to the user yet. Therefore, the tour will appear on the next time the user visits the new "Add product" page. So to enable the tour, we must set  `woocommerce_block_product_tour_shown` to `no`, and that could be done by passing `enable=true` to the `toggleBlockProductTour` function.

Let me know if it still sounds confusing.